scripts/build.sh

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+script_name="${BASH_SOURCE:-$0}"
+script_path=$(realpath "$script_name")
+scripts_dir_path=$(dirname "$script_path")
+
+# shellcheck disable=SC1091
+source "${scripts_dir_path}/lib/_functions.sh"
+# shellcheck disable=SC1091
+source "${scripts_dir_path}/lib/_logger.sh"
+
+function usage {
+    cat <<EOF
+This script performs the following tasks in this specific order:
+1. Generates a list of Operator manifests for a SonataFlow project
+2. Builds the workflow image using 
+3. Optionally deploys.
+    3.1 Pushes the image to a registry (specidied in the image path)
+    3.2 Applies the generated manifests in the current k8s namespace
+
+Usage: 
+    $script_name [flags]
+
+Flags:
+    -i|--image string                 The image path to use for the workflow (required).
```

Comment:
would be nice to have a sample so it's clear what is expected (e.g: quay.io/orchestrator/demo) and that the tag will be set by the script